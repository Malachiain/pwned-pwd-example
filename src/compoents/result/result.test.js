import React, {Component} from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import {Result} from './index'
describe('The result entry screen',()=>{
    it('It renders the result screen correctly',()=>{
        const wrapper = shallow(<Result status="Great" reset={()=>{} } />)
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})