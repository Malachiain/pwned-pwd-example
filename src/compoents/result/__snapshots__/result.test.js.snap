// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The result entry screen It renders the result screen correctly 1`] = `
<styled.div>
  <styled.h3>
    The results are in!
  </styled.h3>
  <styled.div>
    <styled.p>
      Great
    </styled.p>
    <styled.button
      onClick={[Function]}
    >
      Check another password
    </styled.button>
  </styled.div>
</styled.div>
`;
